test/Learners/LearnersTests/Properties/AssemblyInfo.cs

```diff
@@ -10,9 +10,7 @@
 // associated with an assembly.
 [assembly: AssemblyTitle("Microsoft.ML.Probabilistic.Learners.Tests")]
 [assembly: AssemblyDescription("Learner module tests for Infer.NET")]
-[assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("Microsoft Research Limited")]
-[assembly: AssemblyProduct("Microsoft.ML.Probabilistic")]
 [assembly: AssemblyCopyright("Copyright © Microsoft Research Limited 2013")]
```

Comment:
[Not this PR, just a comment] Some of these properties are outdated and would ideally be updated to sane values; where possible, in a single place.

Reply:
I'll check with the team whether I can remove these in a follow up. I agree, but I don't know if these are special values.